.github/workflows/cypress.yml

```diff
@@ -28,6 +28,22 @@ jobs:
           node-version: 16
       - name: Install root dependencies
         run: yarn install
+      - name: Download and unzip latest wallet
+        run: curl -s https://api.github.com/repos/vegaprotocol/vegawallet/releases/latest | awk -F\" '/browser_download_url.*linux-amd64.zip/{print $(NF-1)}' | xargs wget && unzip vegawallet-linux-amd64.zip && rm vegawallet-linux-amd64.zip
```

Comment:
chore (blocking): this is not deterministic. Add a specific version and version configuration field using `matrix` into the above, I have an example if that would help 👍

Reply:
Changed to using matrix for versioning